minhton/docs/using.md

```diff
@@ -144,13 +144,37 @@ Since the node that was previously located at 2:1 moved to 1:1 to replace it, bo
   <figcaption markdown>**Figure 4:** The entries in the routing information of the 4 nodes in the network. The updated entries after the node, which was previously at 1:1 left, are highlighted in blue.</figcaption>
 </figure>
 
-## Entity Search / Peer Discovery
+## Peer Discovery / Entity Search (ESearch)
 
 Each node in the constructed peer-to-peer network may have different capabilities and properties.
 To find peers with specific capabilities and properties, we use a practical solution to the problem of peer discovery, which is finding peers in the network according to a specified query.
 We contribute a peer discovery for an m-ary tree-structured P2P network by utilizing a connected dominating set (CDS), a technique that is typically used in unstructured networks.
+Our solution is also known as Entity Search (ESearch).
 Our approach to constructing the CDS requires no additional communication cost, while nodes can insert, update and remove data within O(1).
 
+Each CDS maintains up to two levels at the same time and consists of a number of dominating set nodes (DSNs).
+Furthermore, each DSN is responsible for several nodes and their KVPs within its cover area (CA) as seen in [Figure 5](#fig_peer_discovery).
+A CA includes, if present, the DSN itself, its *m* RT neighbors to the left and right, its own *m* children, and the routing table children.
```

Comment:
It might improve the reading if the last two sections (esearch and routing links) are swapped, since many link types are already mentioned in the entity search description text. If those sections are swapped, the suggested change above must not be taken